src/github/docs/github-search-api.md

```
@@ -0,0 +1,92 @@
+- Files must be < 384 KB
+- No regex support via API (web UI only)
+- No sort options (best match only)
+- Must include at least one search term (qualifiers alone are insufficient)
+
+> **gh CLI disclaimer:** "Note that these search results are powered by what is now a legacy GitHub code search engine. The results might not match what is seen on github.com."
+
+#### Text Match Metadata
+
+Add header to get highlighted text fragments:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)